extern/agg24-svn/src/platform/mac/agg_platform_support.cpp

```diff
@@ -1041,7 +1041,7 @@ int main(int argc, char* argv[])
 	// argc = ccommand (&argv);
 #endif
     
-    // Check if we are launched by double-clicking under OSX 
+    // Check if we are launched by double-clicking under macOS 
```

Comment:
This should probably be reverted as it is in `extern`.

Reply:
Thanks for catching, I've reverted this change.